libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -449,6 +449,36 @@ string YulUtilFunctions::maskBytesFunctionDynamic()
 	});
 }
 
+string YulUtilFunctions::maskLowerOrderBytesFunction(size_t _bytes)
+{
+	string functionName = "mask_lower_order_bytes_" + to_string(_bytes);
+	solAssert(_bytes <= 32, "");
+	return m_functionCollector.createFunction(functionName, [&]() {
+		return Whiskers(R"(
+			function <functionName>(data) -> result {
+				result := and(data, <mask>)
+			})")
+			("functionName", functionName)
+			("mask", formatNumber(u256(-1) >> (256 - 8 * _bytes)))
```

Comment:
I don't know if we spoke about this, but I was trying to avoid `u256(-1)`

Reply:
Is `~u256(0)` better?

Reply:
Perfect